reference/docs-conceptual/getting-started/fundamental/Learning-PowerShell-Names.md

```diff
@@ -0,0 +1,141 @@
+a computer is `Stop-Computer`. The command to list all computers on a network is `Get-Computer`.
+The command to get the system date is `Get-Date`.
+
+You can list all commands that include a particular verb with the **Verb** parameter for
+`Get-Command`. For example, to see all cmdlets that use the verb `Get`, type:
+
+```
+PS> Get-Command -Verb Get
+
+CommandType     Name                            Definition
+-----------     ----                            ----------
+Cmdlet          Get-Acl                         Get-Acl [[-Path] <String[]>]...
+Cmdlet          Get-Alias                       Get-Alias [[-Name] <String[]...
+Cmdlet          Get-AuthenticodeSignature       Get-AuthenticodeSignature [-...
+Cmdlet          Get-ChildItem                   Get-ChildItem [[-Path] <Stri...
+...
+```
+
+Use the **Noun** parameter to see a family of commands that affect the same type of object. For
+example, run following command to see the commands  available for managing services:
+
+```
+PS> Get-Command -Noun Service
+
+CommandType     Name                            Definition
+-----------     ----                            ----------
+Cmdlet          Get-Service                     Get-Service [[-Name] <String...
+Cmdlet          New-Service                     New-Service [-Name] <String>...
+Cmdlet          Restart-Service                 Restart-Service [-Name] <Str...
+Cmdlet          Resume-Service                  Resume-Service [-Name] <Stri...
+Cmdlet          Set-Service                     Set-Service [-Name] <String>...
+Cmdlet          Start-Service                   Start-Service [-Name] <Strin...
+Cmdlet          Stop-Service                    Stop-Service [-Name] <String...
+Cmdlet          Suspend-Service                 Suspend-Service [-Name] <Str...
+...
+```
+
+## Cmdlets use standard parameters
+
+As noted earlier, commands used in traditional command-line interfaces don't always have consistent
+parameter names. Parameters are often single-character or abbreviated words that are easy to type
+but aren't easily understood by new users.
+
+Unlike most other traditional command-line interfaces, PowerShell processes parameters directly,
+and it uses this direct access to the parameters along with developer guidance to standardize
+parameter names. This guidance encourages but does not guarantee that every cmdlet conforms to the standard.
+
+PowerShell also standardizes the parameter separator. Parameter names always have a '-' prepended to them with a PowerShell command. Consider the following example:
+
+```powershell
+Get-Command -Name Clear-Host
+```
+
+The parameter's name is **Name**, but it is typed as `-Name` when used on the command line as a
+parameter.
+
+Here are some of the general characteristics of the standard parameter names and usages.
+
+### The **Help** parameter (?)
+
+When you specify the `-Help` or `-?` parameter on any cmdlet, PowerShell displays help for the
```

Comment:
Parameters should be bold, not code-fenced.

Reply:
We need to update the contributor guide.